docs/niscope/functions.rst

```diff
@@ -110,7 +110,7 @@ niscope.Session methods
 
 
 
-.. function:: cal_self_calibrate(option=niscope.Option.SELF_CALIBRATE_ALL_CHANNELS)
+.. function:: cal_self_calibrate(option=niscope.enums.Option.SELF_CALIBRATE_ALL_CHANNELS)
```

Comment:
Is this a fix to the documentation?
I thought when you import niscope all the enums would be imported inside the niscope namespace.

Is that not the case? And if not, isn't that what we'd want?
I couldn't find "enums" in any of the examples.

```
Kirsch0:nimi-python kirsch$ find  * | grep examples.*\.py | xargs grep -ir enums
```

Reply:
I had simplified add_all too much. I can't put the 'enum.' in functions_addon and instead have to go back to enums being a special case for default values.